src/createRoom.ts

```diff
@@ -168,6 +168,15 @@ export default async function createRoom(opts: IOpts): Promise<string | null> {
                 },
             };
         }
+    } else {
+        createOpts.power_level_content_override = {
+            events: {
+                // Element Call should be disabled by default
+                [ElementCall.MEMBER_EVENT_TYPE.name]: 100,
+                // Make sure only admins can enable it
+                [ElementCall.CALL_EVENT_TYPE.name]: 100,
```

Comment:
I would expect these power levels to match the levels you get when the switch is enabled - since we want to enable Element Call in new rooms by default

Reply:
(https://www.figma.com/file/STqNwYb2rCXxHAAku2JQA7/Element-Call-Integration?node-id=615%3A44489)

Reply:
Oh, I see